### ISSUE TITLE
Question: Can we access this library in swift side?

### ISSUE THREAD

**Issue by nis-ship-it** [russhwolf/multiplatform-settings#214]
At the moment, I'm working on integrating this library in kmp project and it's working great on android side with no args module. The instance is created directly in koin under the common folder however issue is that I'm not able to access "Setting" dataType in swift code. 

Koin is already working in ios side so no issues with that but it can find the settings definition or data type. 

Do you know anything on this? Would I have to use datastore for this section?

**Comment by russhwolf** [comment]
Yes, it's possible. Generally, dependencies only export from a Kotlin framework if they leak into the public API. For example, you might have a public function that includes a Koin type in its signature, but not one that includes a Multiplatform Settings type. So you can add such a function, or you can export the whole library as described [here](https://kotlinlang.org/docs/multiplatform-build-native-binaries.html#export-dependencies-to-binaries)

**Comment by russhwolf** [comment]
Are you still having trouble here? If not, I'm inclined to close this issue.
